pytorch/training/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -486,7 +486,8 @@ RUN pip install --no-cache-dir -U \
     "sagemaker>=2,<3" \
     sagemaker-experiments==0.* \
     sagemaker-pytorch-training \
-    flash-attn==0.2.6.post1
+    flash-attn==0.2.8 \
+    triton==2.0.0.dev20221202
```

Comment:
Why do we need triton?

Reply:
Although openai's triton is directly depended by PyTorch, Rubik team currently requires a specific build of triton, hence the explicit installation.

Reply:
@ShiboXing Please define the Triton version required as an ARG next to the ARG that defines the SMMP binary if these are both interrelated packages.

Also, please add a comment mentioning that the Triton version installed in the image is dependent on the SMMP version.